app/routes/missions.svom/route.mdx

```diff
@@ -0,0 +1,67 @@
+---
+handle:
+  breadcrumb: SVOM
+---
+
+import logo from './logo.png'
+
+# Space Variable Objects Monitor (SVOM)
+
+<img
+  src={logo}
+  align="right"
+  width="250"
+  height="165"
+  alt="SVOM logo"
+  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
+/>
+
+**Launch Date:** June 22, 2024
+
+**Anticipated Start of Science Operations:** September 2024
+
+**End of Operations:** Mission duration of 3 years
+
+**Data Archive:**
+[HEASARC SVOM Archive](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/svom.html)
+
+The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://irfu.cea.fr/dap/en/Phocea/Vie_des_labos/Ast/ast_technique.php?id_ast=2276). SVOM mission is dedicated to the study of the most distant explosions of stars, the gamma-ray bursts (GRBs).
+
+The SVOM spacecraft carries 4 main instruments of which 2 are French (ECLAIRs and MXT) and 2 are Chinese (GRM and VT). Transient alert messages will be sent for:
+
+ <div className="overflow-table">
+
+| Instruments                        | Energy Range            | Field of View | Localization                                   |
+| ---------------------------------- | ----------------------- | ------------- | ---------------------------------------------- |
+| ECLAIRs                            | 4 - 250 keV             | 2 ster        | ~8' radius (statistical, 90% confidence)       |
+| Gamma Ray burst Monitor (GRM)      | 15 - 5000 keV           | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
+| Microchannel X-ray Telescope (MXT) | 15 - 5000 keV           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
+| Visual Telescope (VT)              | Blue Band: 450 - 650 nm | 26' x 26'     | ~TBD'' radius (statistical, 90% confidence)    |
+|                                    | Red Band: 650 - 1000 nm |               |                                                |
```

Comment:
This needs to also have the VT details, or be on the same row in the table as the blue info.